src/dos/dos.cpp

```diff
@@ -365,13 +365,7 @@ static Bitu DOS_21Handler(void) {
 			break;
 		};
 		break;
-	case 0x07:		/* Character Input, without echo */
-		{
-				uint8_t c;uint16_t n=1;
-				DOS_ReadFile (STDIN,&c,&n);
-				reg_al=c;
-				break;
-		};
+	case 0x07:              /* Character Input, without echo */
```

Comment:
Seems like a change in functionality. Was this intentional?

Reply:
Just de-duping to fix a PVS studio error
(This 0x07 case is now adjacent to 0x08, as they have the same code body).